src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -312,6 +312,68 @@ newrelic-infrastructure:
   Setting `interval` to values larger than `40s` is not allowed.
 </Callout>
 
+### Filtering Namespaces [#filter-namespace]
+
+The Kubernetes Integration v3 and above allows filtering which namespaces will be scraped by allowing you to include or exclude namespaces based on labels. By default all namespaces will be scraped.
+
+In order to include only namespaces matching a certain label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infratructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        newrelic.com/scrape: true
+```
+
+In this example only namespaces with the label `newrelic.com/scrape` set to true will be scraped. You can also use you own custom label:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      namespaceSelector:
+        matchLabels:
+          newrelic.com/scrape: true
+```
+
+To exclude certain namespaces use the following syntax:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchExpressions:
+      - {key: newrelic.com/scrape, operator: NotIn, values: [false]}
```

Comment:
I'll mention that the operator `In` can be used and if more than a expression is provided they are "anded".  In the end we're making a call to https://pkg.go.dev/k8s.io/apimachinery/pkg/labels#Parse for each `matchExpression`.